server/services/assessRisksAndNeedsService.ts

```diff
@@ -6,13 +6,16 @@ import { SupplementaryRiskInformation } from '../models/assessRisksAndNeeds/supp
 export default class AssessRisksAndNeedsService {
   constructor(private readonly hmppsAuthService: HmppsAuthService, private readonly restClient: RestClient) {}
 
-  async getSupplementaryRiskInformation(riskId: string): Promise<SupplementaryRiskInformation> {
-    const token = await this.hmppsAuthService.getApiClientToken()
-
+  async getSupplementaryRiskInformation(
+    riskId: string,
+    token: string | null = null
+  ): Promise<SupplementaryRiskInformation> {
     logger.info({ riskId }, 'getting supplementary risk information')
     return (await this.restClient.get({
       path: `/risks/supplementary/${riskId}`,
-      token,
+      token: token || (await this.hmppsAuthService.getApiClientToken()),
```

Comment:
Am I missing something here? What's this change for? Is it to do with the 2 different types of credentials? Would you mind adding a bit of a description to the commit message as to why it's necessary?

Reply:
yeh there are some changes coming to ARN to use the user tokens for all the endpoints. will mention in commit message.